js/ui/date_range_box.js

```diff
@@ -0,0 +1,33 @@
+import registerComponent from '../core/component_registrator';
+import { extend } from '../core/utils/extend';
+import DateBox from './date_box';
+
+const DateRangeBox = DateBox.inherit({
```

Comment:
May be we start write component as class component?
`class DateRangeBox extends DateBox { ...`

Reply:
I'm not sure it won't cause any issues, but let's try